src/paymentservice/server.js

```diff
@@ -55,17 +55,25 @@ class HipsterShopServer {
     }
   }
 
-  static CheckHandler (call, callback) {
```

Comment:
Thought: I think we can avoid this type of noise in pull-requests in the future by adding linting and automation to maintain consistent code style across this repository. This is something Olivier and I have been exploring for our 2022 roadmap.